examples/misc_exporter_gltf.html

```diff
@@ -48,13 +49,22 @@
 				var options = {
 					trs: document.getElementById('option_trs').checked,
 					onlyVisible: document.getElementById('option_visible').checked,
-					truncateDrawRange: document.getElementById('option_drawrange').checked
-				}
+					truncateDrawRange: document.getElementById('option_drawrange').checked,
+					binary: document.getElementById('option_binary').checked
+				};
 				gltfExporter.parse( input, function( result ) {
 
-					var output = JSON.stringify( result, null, 2 );
-					console.log( output );
-					saveString( output, 'scene.gltf' );
+					if ( result instanceof ArrayBuffer ) {
+
+						saveArrayBuffer( result, 'scene.glb' );
+
+					} else {
+
+						var output = JSON.stringify( result, null, 2 );
+						console.log( output );
+						saveString( output, 'scene.gltf' );
```

Comment:
Maybe modifying the extension to `glb` if binary is selected?

Reply:
^Handled above this section